pages/_app.js

```diff
@@ -1,4 +1,4 @@
-import styles from '../styles/global.css'
+import '../styles/global.css'
```

Comment:
What's the reason for this change? Tidier formatting?

Reply:
I don't think the former imported the style sheet correctly.